docs/_articles/en/user-guide/org-browser.md

```diff
@@ -25,7 +25,11 @@ When you open the Org Browser, all the metadata for the default org is saved in
 
 ## Listing and Refreshing Metadata
 
-The Org Browser lists all the metadata types **(4)** in your default org. To view the components **(5)**, expand the metadata type. When you expand metadata types with folders such as Reports, Dashboards, Documents, and EmailTemplates, all the available folders are displayed. You can view the components in a folder by expanding it.
+The Org Browser lists all the metadata types **(4)** in your default org. To view the components **(5)**, expand the metadata type. When you expand metadata types with folders such as Reports, Dashboards, Documents, and EmailTemplates, all the available folders are displayed. You can view the components in a folder by expanding it. Expand the Custom Object metadata type to view fields and their type information. Additional information is displayed for following field types:
+-  strings: string(length)
+-  textarea: textarea(length)
+-  email: email(length)
+-  Lookup: lookup(reference)
```

Comment:
Is "Lookup" capitalized?

Reply:
"Lookup" is more of an alias, for when an object has a foreign-key relationship with another object.  For example, the `Campaign` object has `Parent (reference)` and `Owner (reference)`, so my suggestion would be to change the docs to `lookup: lookup(reference)`.  Also, we should probably pluralize each:
```
-  strings: string(length)
-  textareas: textarea(length)
-  emails: email(length)
-  lookups: lookup(reference)
```

Reply:
Thanks @jeffb-sfdc for the explanation!